packages/@expo/cli/src/utils/env.ts

```diff
@@ -153,6 +153,14 @@ class Env {
   get EXPO_USE_CUSTOM_INSPECTOR_PROXY(): boolean {
     return boolish('EXPO_USE_CUSTOM_INSPECTOR_PROXY', false);
   }
+
+  /** Enable Expo Router typed router generation */
+  get EXPO_ROUTER_TYPED_ROUTES() {
+    return boolish('EXPO_ROUTER_TYPED_ROUTES', false);
+  }
+  get EXPO_ROUTER_APP_ROOT() {
+    return string('EXPO_ROUTER_APP_ROOT', '../../app');
```

Comment:
This cannot have a default value, it should always be calculated to ensure the root is correctly relative to the source entry.